api/v1beta2/awsmachine_webhook.go

```diff
@@ -386,6 +386,9 @@ func (r *AWSMachine) validateCapacityReservation() field.ErrorList {
 	if r.Spec.CapacityReservationID != nil && r.Spec.CapacityReservationPreference != CapacityReservationPreferenceOnly && r.Spec.CapacityReservationPreference != "" {
 		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec", "capacityReservationPreference"), "when a reservation ID is specified, capacityReservationPreference may only be `capacity-reservations-only` or empty"))
```

Comment:
Could you also change this please, thanks!
```suggestion
		allErrs = append(allErrs, field.Forbidden(field.NewPath("spec", "capacityReservationPreference"), "when a capacityReservationID is specified, capacityReservationPreference may only be `CapacityReservationsOnly` or empty"))
```